### PR TITLE
Additional changes for Drill Metastore docs

### PR DESCRIPTION
**Pull request by vvysotskyi**
Changes after code review for https://github.com/apache/drill/pull/1953

### CODE REVIEW

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ The Metastore is a Beta feature; it is subject to change. We encourage you to tr
...
+One of the main advantages of Drill is schema-on-read. But Drill can’t handle some cases with this approach, there are the issues related to Schema Evolution and Schema Changes.
+
+Significant benefits of schema-aware execution:
+
+ - At Planning time:
+    - Better scope for planning optimizations.
+    - Proper estimation of column widths since types are known, hence more accurate costing.
+    - Graceful early exit if certain data type validations fail.
+ - At Runtime:
+    - Avoids some cases with `SchemaChange` exceptions. All minor fragments will have a common understanding of the schema.
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]: Use noun to comply with other points (scope, estimation, exit): SchemaChange exception avoidance

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 2]
```diff
@@ -48,42 +83,52 @@ drill.metastore: {
...
-is storage plugin, workspace, table name, and metadata key.
+A traditional database divides tables into schemas and tables.
+Drill can connect to any number of data sources, each of which may have its own schema.
+As a result, the Metastore labels tables with a combination of (plugin configuration name, workspace name, table name).
+Note that if before renaming any of these items, you must delete table's Metadata entry and recreate it after renaming.
 
 ### Related Session/System Options
 
-The following options are set via `ALTER SYSTEM SET`, or `ALTER SESSION SET` or via the Drill Web console.
+The metastore provides a number of options to fit your environment. The default options are find in most cases.
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]:
```suggestion
The Metastore provides a number of options to fit your environment. The default options are find in most cases.
```

Comment (arina-ielchiieva) [reply _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]: `are find in most cases.`?

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 3]
```diff
@@ -48,42 +83,52 @@ drill.metastore: {
...
+The options are set via `ALTER SYSTEM SET`, `ALTER SESSION SET` or the Drill Web console.
+
+In general, you should set the options via `ALTER SYSTEM` so that they take effect for all users.
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]: But this only admin can do...

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 4]
```diff
@@ -160,19 +199,78 @@ Drill uses the Metastore in several places. When you run a query with multiple d
  then Drill will assume that existing metadata is invalid and wouldn't use it. Periodically rerun `ANALYZE TABLE` so
  that Drill can use table metadata when possible.
 
-### Limitations
+### Exposing Drill Metastore metadata through `INFORMATION_SCHEMA` tables
+
+Drill exposes some Metastore tables metadata through `INFORMATION_SCHEMA` tables.
+Note, that Metastore metadata will be exposed to the info schema, only if Metastore is enabled. If it is disabled, info
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]:
```suggestion
Note, that Metastore metadata will be exposed to the `INFORMATION_SCHEMA`, only if Metastore is enabled. If it is disabled, info
```

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md [hunk 5]
```diff
@@ -16,21 +16,25 @@ If the file system does not support atomic rename, it could lead to inconsistenc
 
 ### Iceberg Tables Location
 
-Iceberg tables will reside on the file system in the location based on
-Iceberg Metastore base location `drill.metastore.iceberg.location.base_path` and component specific location.
-If Iceberg Metastore base location is `/drill/metastore/iceberg`
-and tables component location is `tables`. Iceberg table for tables component
-will be located in `/drill/metastore/iceberg/tables` folder.
+Iceberg is essentially a file system within a file. The Iceberg table is stored in a file system. In the tutorial, we
+ stored the metastore in your local file system. Drill is a distributed query engine, so production deployments store
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]:
```suggestion
 stored the Metastore in your local file system. Drill is a distributed query engine, so production deployments MUST store
```

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md [hunk 6]
```diff
@@ -16,21 +16,25 @@ If the file system does not support atomic rename, it could lead to inconsistenc
...
+ the metastore in your DFS such as HDFS.
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]:
```suggestion
 the Metastore on DFS such as HDFS.
```

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md [hunk 7]
```diff
@@ -16,21 +16,25 @@ If the file system does not support atomic rename, it could lead to inconsistenc
...
-Assume Iceberg table location is `/drill/metastore/iceberg/tables`, metadata for the table
-`dfs.tmp.nation` will be stored in the `/drill/metastore/iceberg/tables/dfs/tmp/nation` folder.
+Within the Metastore directory, the Metastore stores the data for each table as a set of records within a single Iceberg table.
 
-Example of base Metastore configuration file `drill-metastore-override.conf`, where Iceberg tables will be stored in
+Iceberg tables will reside on the file system in the location based on Iceberg Metastore base location
+ `drill.metastore.iceberg.location.base_path` configuration property and component specific location.
+Table metadata resides in the `${drill.metastore.iceberg.location.base_path}/tables` directory.
+
+Within the tables directory, there is one file per analyzed table in the form `my-dfs/my-workspace/table-name/UUID.parquet`
```
Comment (arina-ielchiieva) [inline _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]: Not sure about meaning of this sentence.

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 8]
```diff
@@ -364,7 +406,10 @@ apache drill> ANALYZE TABLE dfs.tmp.lineitem DROP METADATA;
 1 row selected (0.291 seconds)
 ```
 
-### Collecting metadata for specific set of columns
+### Collect metadata for specific set of columns
+
+Next lets gather metadata for a subset of the columns in the `lineitem` table. You would do this when you know
+ which columns will be present in filter conditions or data size or columns count is too large.
```
Comment (paul-rogers) [inline _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]: A bit awkward. Maybe something like: "You would do this to allow Drill to optimize `WHERE` conditions on certain columns. Also, if file size or the number of columns grows large, it can take too long to gather all statistics. Instead you can speed up analysis by gathering statistics only for selected columns: those actually used in the `WHERE` clause."

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md [hunk 9]
```diff
@@ -16,21 +16,25 @@ If the file system does not support atomic rename, it could lead to inconsistenc
 
 ### Iceberg Tables Location
 
-Iceberg tables will reside on the file system in the location based on
-Iceberg Metastore base location `drill.metastore.iceberg.location.base_path` and component specific location.
-If Iceberg Metastore base location is `/drill/metastore/iceberg`
-and tables component location is `tables`. Iceberg table for tables component
-will be located in `/drill/metastore/iceberg/tables` folder.
+Iceberg is essentially a file system within a file. The Iceberg table is stored in a file system. In the tutorial, we
```
Comment (paul-rogers) [inline _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]: Remove comma: "In the tutorial we..."

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md (hunk 7 above)
Comment (paul-rogers) [reply _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]: Maybe reword to: "The tables directory contains one file per analyzed table. Each file has a name of the form: ..." Question? Why does the file have a ".parquet" suffix if it is an Iceberg file? I get that the actual data file is in Parquet format. But, this file is not Parquet, it is a Iceberg file that describes a Parquet file, correct? This is likely to be confusing to more than just me.

File: _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md [hunk 10]
```diff
@@ -16,21 +16,25 @@ If the file system does not support atomic rename, it could lead to inconsistenc
...
 
-Example of base Metastore configuration file `drill-metastore-override.conf`, where Iceberg tables will be stored in
+Iceberg tables will reside on the file system in the location based on Iceberg Metastore base location
+ `drill.metastore.iceberg.location.base_path` configuration property and component specific location.
+Table metadata resides in the `${drill.metastore.iceberg.location.base_path}/tables` directory.
+
+Within the tables directory, there is one file per analyzed table in the form `my-dfs/my-workspace/table-name/UUID.parquet`
+
+If you ran through the [Tutorial]({{site.baseurl}}/docs/using-drill-metastore/#tutorial), Metastore files are stored in `/home/username/drill/metastore/iceberg/tables`.
+If you inspect this directory, you will see the following directories `my-dfs/tmp/lineitem/`.
```
Comment (paul-rogers) [inline _docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md]: Remove comma: "...inspect this directory you will..."

File: _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md [hunk 11]
```diff
@@ -1,35 +1,18 @@
...
- query performance. In general, large tables benefit from statistics more than small tables do.
-
-Unlike Hive, Drill does not require you to declare a schema. Instead, Drill infers the schema by scanning your table 
- and computes some metadata like MIN / MAX column values and NULLS COUNT designated as "metadata" to be able to
- produce more optimizations like filter push-down, etc. If `planner.statistics.use` option is enabled, this command
- will also calculate and store table statistics into Drill Metastore.
-
-Unlike Hive, Drill does not require you to declare a schema. Instead, Drill infers the schema by scanning your table
- in the same way as it is done during regular select.
+Please refer to [Using Drill Metastore]({{site.baseurl}}/docs/using-drill-metastore) for more details about Drill Metastore, its purpose and usage information.
```
Comment (paul-rogers) [inline _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md]: Perhaps: "...about Drill Metastore including its purpose and how to use it."

Comment (vvysotskyi) [reply _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md]: Thanks, reworded.

File: _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md [hunk 12]
```diff
@@ -10,6 +10,31 @@ The Metastore is a Beta feature; it is subject to change. We encourage you to tr
 Because the Metastore is in Beta, the SQL commands and Metastore formats may change in the next release.
 {% include startnote.html %}In Drill 1.17, this feature is supported for Parquet tables only and is disabled by default.{% include endnote.html %}
 
+## Drill Metastore introduction
+
+One of the main advantages of Drill is schema-on-read. But Drill can’t handle some cases with this approach, there are the issues related to Schema Evolution and Schema Changes.
```
[the remark this answers is not in the document]
Comment (vvysotskyi) [reply _docs/performance-tuning/drill-metastore/010-using-drill-metastore.md]: Thanks, removed capitalization here and replaced schema changes with ambiguous schema.

This is definitely useful info about the ambiguous schema and how and why Drill falls into this data trap, but I think it should be placed on a separate page, not in the Drill Metastore introduction.

File: _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md [hunk 13]
```diff
@@ -29,6 +30,10 @@ The ANALYZE TABLE REFRESH METADATA statement supports the following syntax:
 The name of the table or directory for which Drill will collect table metadata. If the table does not exist, the table
  is temporary or if you do not have permission to read the table, the command fails and metadata is not collected and stored.
 
+*table({table function name}(parameters))*
```
Comment (arina-ielchiieva) [inline _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md]: Well, first of all this syntax is only available since Drill 1.18.
Secondly, we do need some example for the users.

Comment (vvysotskyi) [reply _docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md]: Thanks, added an example and specified the version where it is available.

**Comment by arina-ielchiieva** [comment]
+1 from my side, @paul-rogers any other comments?
